### PR TITLE
Feature2

### PR DESCRIPTION
**Pull request by Leandro-Barretoo**
In this PR I changed the table, adding a column;
Added more data to it;
Used transaction methods and finished with some queries.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -31,3 +31,20 @@ WHERE name NOT IN ('Gabumon');
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts > 0;
```
Comment (oxenprogrammer) [inline queries.sql]: **Changes Required**
- [x] We need to select where animals have `never` escaped. Apparently this query returns animals that have escaped

File: queries.sql [hunk 2]
```diff
@@ -31,3 +31,20 @@ WHERE name NOT IN ('Gabumon');
...
+
+SELECT AVG(weight_kg) FROM animals;
+
```
Comment (oxenprogrammer) [inline queries.sql:41]: **Changes Required**
- [x] I don't seem to see the query for `Who escapes the most, neutered or not neutered animals?` Let us please add it

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @Leandro-Barretoo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 3]
```diff
@@ -31,3 +31,20 @@ WHERE name NOT IN ('Gabumon');
...
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT SUM(escape_attempts) FROM animals
+GROUP BY neutered;
```
Comment (Leandro-Barretoo) [inline queries.sql:43]: This is the query for `Who escapes the most, neutered or not neutered animals?`

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉✔**
Hello @Leandro-Barretoo 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**